test/views/declaration/AdditionalInformationViewSpec.scala

```diff
@@ -124,6 +124,18 @@ class AdditionalInformationViewSpec extends UnitViewSpec with ExportsTestData wi
             .get(1) must containMessage("declaration.additionalInformation.table.headers.description")
         }
 
+        "has visually hidden Change Link header" in {
+          view
+            .select("#additional_information thead tr th")
+            .get(2) must containMessage("site.change.header")
```

Comment:
:+1 This is the preferred way of checking text - `element must containMessage("message.key")`
There is also a `containMessageForElements` version where you are checking Elements (not Element)